r/R/dplyr-funcs-datetime.R

```diff
@@ -429,6 +430,78 @@ register_bindings_datetime_conversion <- function() {
   })
 }
 
+register_bindings_datetime_timezone <- function() {
+  register_binding(
+    "lubridate::force_tz",
+    function(time, tzone = "", roll_dst = c("error", "post")) {
+      if (length(roll_dst) == 1L) {
+        roll_dst <- c(roll_dst, roll_dst)
+      } else if (length(roll_dst) != 2L) {
+        arrow_not_supported("`roll_dst` length != 1 or 2")
```

Comment:
```suggestion
        arrow_not_supported("`roll_dst` must be 1 or 2 items long; other lengths")
```
This phrasing could be a bit clearer here - I've updated this to be more descriptive, and use "must" phrasing, as per the tidyverse style guide.